src/Bicep.Core/Emit/TemplateWriter.cs

```diff
@@ -30,7 +31,7 @@ public class TemplateWriter : ITemplateWriter
         public const string NestedDeploymentResourceType = AzResourceTypeProvider.ResourceTypeDeployments;
         
         // IMPORTANT: Do not update this API version until the new one is confirmed to be deployed and available in ALL the clouds.
-        public const string NestedDeploymentResourceApiVersion = "2019-10-01";
+        public const string NestedDeploymentResourceApiVersion = "2020-06-01";
```

Comment:
>NestedDeploymentResourceApiVersion [](http://example.com/codeflow?start=28&length=34)

We should probably go through the ARM code and see if we made any changes between `2019-10-01` and `2020-06-01` that could impact Bicep.

Separately from that, is `2020-06-01` available in all the clouds?

@alex-frankel We should probably call this out explicitly in the release notes for v0.5

Reply:
`2020-06-01` added `id` and `relativePath` properties added to `TemplateLink` object to support template specs. There are two other API versions `2019-11-01` and `2019-11-01` newer than `2019-10-01`, but they only contain changes for `/subscriptions`.

`2020-06-01` should be available in all the clouds as Template Spec is GA, plus I confirmed that `2021-01-01` was deployed to all clouds for What-If, so we should be fine.

Reply:
Sounds good.

Reply:
If 2021-01-01 is live then why not use it?

Reply:
We could, but it's not necessary as it only contains changes for what-if.